### DISCUSSION TITLE
BrainModel.h5 missing

### DISCUSSION THREAD

**Discussion by datacrud8** [KalyanM45/OpenHealth#7] [General]
Hello, I like this project and trying to build it locally.
After a successful install , when I run the "python app.py" it fails with the below error.
"
OSError: No file or directory found at Artifacts\Brain_Tumour\BrainModel.h5
OSError: No file or directory found at Artifacts\Kidney_Disease\Kidney_Model.h5
"
Do we need to manually download these h5 files (BrainModel.h5 and Kidney_Model.h5) ? Please share the file path.

Also just for testing , removed these two lines from app.py
brain_model = load_model('Artifacts\Brain_Tumour\BrainModel.h5')
kidney_model = load_model('Artifacts\Kidney_Disease\Kidney_Model.h5')

and page loading in localhost:5000 but none of the links are working.
Do we need to add any other configuration? please confirm.

Thanks

[a comment by an automated account is not shown]

**Comment by KalyanM45** [comment]
You need to train the model, As the complete project is in the development stage, we are working on another model but as this model is big, I didn't track it in this version. Better implement it on your own or else mail me ill send you the model files.
